Suggest some changes in all-map-locations (#803)
Heegu-sama/Homm3BG: sections/all_map_locations.tex

\columnbreak

\textbf{Settlements}\index{Settlement} act as a spawn point for Secondary Heroes, and as a place for Main Heroes to move to when defeated.
When you Flag a Settlement, choose one of the following options: 
\begin{itemize} 

> 	\item Increase your \svg{gold}, \svg{building_materials} or \svg{valuablegreater} income by one space. As with Mines, if you are the first player to Flag a Settlement, you \textbf{immediately} gain Resources equal to that increase in production. Mark the Settlement with an appropriate Resource Token to show which Resource it produces. When you Flag an enemy Settlement, you may change this Resource.

Reviewer: The "tab" symbol at the start caused the failure

> 	\item Increase your \svg{gold}, \svg{building_materials} or \svg{valuablegreater} income by one space. As with Mines, if you are the first player to Flag a Settlement, you \textbf{immediately} gain Resources equal to that increase in production. Mark the Settlement with an appropriate Resource Token to show which Resource it produces. When you Flag an enemy Settlement, you may change this Resource.

Reviewer 2: ```suggestion
  \item Increase your \svg{gold}, \svg{building_materials} or \svg{valuablegreater} income by one space.
    As with Mines, if you are the first player to Flag a Settlement, you \textbf{immediately} gain Resources equal to that increase in production.
      Mark the Settlement with an appropriate Resource Token to show which Resource it produces.
      When you Flag an enemy Settlement, you may change this Resource.
```
I suggest sticking with "one sentence per line" convention.

> 	\item \textbf{instead of increasing Resource Production}, you may choose to \textbf{Reinforce} one of your \svgunit{bronze} or \svgunit{silver} Units immediately for half the normal cost, rounded up. If you were the first player to Flag the Settlement, Reinforce that Unit \textbf{for free} instead. Do not place any Resource Tokens on the Settlement if you choose to Reinforce.

Reviewer 2: ```suggestion
  \item \textbf{instead of increasing Resource Production}, you may choose to \textbf{Reinforce} one of your \svgunit{bronze} or \svgunit{silver} Units immediately for half the normal cost, rounded up.
    If you were the first player to Flag the Settlement, Reinforce that Unit \textbf{for free} instead.
    Do not place any Resource Tokens on the Settlement if you choose to Reinforce.
```

    \pagetarget{Trading Post}{\textbf{Trading Post}}\par
    \framedimage[\textwidth]{\map_locations/trading_post.jpg}
    \caption{\small Category: \textbf{Revisitable}\\

>       \textbf{Choose one}: \protect\pagelink{Trading}{Trade} resources OR Remove one card and gain 1 \svg{gold} OR buy a \protect\pagelink{War Machines}{War Machine} at the higher cost.

Reviewer 2: ```suggestion
      \textbf{Choose one}: \protect\pagelink{Trading}{Trade} resources OR Remove one card and gain 1 \svg{gold} OR buy a \protect\pagelink{War Machines}{War Machine} at a higher cost.
```

    }
  \end{minipage}\hfill
  \begin{minipage}[t]{0.47\textwidth}
    \vspace{0pt}
    \centering
    \phantom{j}\pagetarget{War Machine Factory}{\textbf{War Machine Factory}}\par
    \framedimage[\textwidth]{\map_locations/war_machine_factory.jpg}

>     \caption{\small Category: \textbf{Revisitable}\\Buy a \protect\pagelink{War Machines}{War Machine} at the cheaper cost.\phantom{.......}}

Reviewer 2: ```suggestion
    \caption{\small Category: \textbf{Revisitable}\\Buy a \protect\pagelink{War Machines}{War Machine} at a lower cost.\phantom{.......}}
```

    \pagetarget{Trading Post}{\textbf{Trading Post}}\par
    \framedimage[\textwidth]{\map_locations/trading_post.jpg}
    \caption{\small Category: \textbf{Revisitable}\\

>       \textbf{Choose one}: \protect\pagelink{Trading}{Trade} resources OR Remove one card and gain 1 \svg{gold} OR buy a \protect\pagelink{War Machines}{War Machine} at a higher cost.

Reviewer 2: Shouldn't removing be a hyperlink too? Because you can't just remove any card.

Author: It would be better. What have i to do in the latex-code to create a link to one specific text passage? I only know how to add links to headlines.

But apart from that we had then three hyperlinks. Probably one Hyperlink to trading page is enough and we use a wording like @DarthGandalf suggested: "for Details see ...".

What do you prefer?

Reviewer 2: I don't know, I guess we have hyperlinks so that we don't need "for details, see...". Maybe it's not needed, since removing is described just below the hypertarget for trading. It would refer just a couple of lines lower.

    \centering
    \textbf{Redwood Observatory}\par
    \framedimage[\linewidth]{\map_locations/redwood_observatory.jpg}

>     \caption{\small Category: \textbf{Visitable}\\Discover a face down Tile adjacent to this one OR \pagelink{Placing}{place} a new Tile from your supply adjacent to this and a further tile.}

Reviewer 2: ```suggestion
    \caption{\small Category: \textbf{Visitable}\\Discover a face down Tile adjacent to this one OR \pagelink{Placing}{place} a new Tile from your pool adjacent to this one.}
```
Placing means adjacent to 2 tiles, so isn't it redundant?
